index.xml

```diff
@@ -346,7 +347,8 @@ binary string, which is then base 64 encoded and placed into the
 In order to verify a signature, a server MUST:
     <list style="numbers">
      <t>
-Use the received HTTP message, the `headers` value, and the Signature String
+Use the received HTTP message, the `headers` value
+(or the Date header if no `headers` value is supplied), and the Signature String
```

Comment:
What happens if `headers=""`?

Reply:
Clarified language in 68d73e8548c3413aa7624a4a9ef6914d9fc55143 to refer to ``parameter`` rather than ``value``.

That said, I am working on clarification for this case and empty header values in response to #41